README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/gnosis/safe-config-service/badge.svg)](https://coveralls.io/github/gnosis/safe-config-service)
 
-The `safe-config-service` is a service that provides configuration information in the context of the Safe clients environment (eg.: list of available safe apps).
+The `safe-config-service` is a service that provides configuration information in the context of the Safe clients environment (eg.: list of available safe apps, or chain metadata).
```

Comment:
nitpick: I would say `and` chain metadata

Reply:
e3e4c27